DirectProgramming/DPC++FPGA/Tutorials/Features/ac_fixed/src/ac_fixed.cpp

```diff
@@ -0,0 +1,200 @@
+// clang-format off
+#include <CL/sycl.hpp>
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/ext/intel/ac_types/ac_fixed.hpp>
+#include <sycl/ext/intel/ac_types/ac_fixed_math.hpp>
+// clang-format on
+
+// dpc_common.hpp can be found in the dev-utilities include folder.
+// e.g., $ONEAPI_ROOT/dev-utilities//include/dpc_common.hpp
+#include "dpc_common.hpp"
+
+using namespace sycl;
+
+using fixed_10_3_t = ac_fixed<10, 3, true>;
+using fixed_9_2_t = ac_fixed<9, 2, true>;
+
+// Forward declare the kernel name in the global scope.
+// This is a FPGA best practice that reduces name mangling in the reports.
+class ConstructFromFloat;
+class ConstructFromACFixed;
+class CalculateWithFloat;
+class CalculateWithACFixed;
+
+// Not recommended Usage example:
+// Convert dynamic float value inside the kernel
+void TestConstructFromFloat(queue &q, const float &a,
+                            ac_fixed<20, 10, true, AC_RND, AC_SAT> &b) {
+  buffer<float, 1> inp_buffer(&a, 1);
+  buffer<ac_fixed<20, 10, true, AC_RND, AC_SAT>, 1> ret_buffer(&b, 1);
```

Comment:
You made two type aliases above (fixed_10_3_t and fixed_9_2_t), why don't you have a third one with this datatype? It could also be a good to explain in this unique location what AC_RND and AC_SAT parameters are doing